base/src/MotionVectorExtractor.cpp

```diff
@@ -202,6 +208,19 @@ int DetailFfmpeg::decodeAndGetMotionVectors(AVPacket* pkt, frame_container& fram
 			{
 				outFrame = makeFrameWithPinId(0, motionVectorPinId);
 			}
+			if (sendOverlayFrame == false) {
```

Comment:
1. formatting
2. put a comment on what is this change about

Reply:
Done